applications/applications_interface.py

```diff
@@ -44,6 +44,8 @@
 Import_MultiScaleApplication = False
```

Comment:
applications_interface is no longer used, we have it here for backwards compatibility (and I think we should remove it soon), but you don't need to modify it.